### PR TITLE
Add flake8-bugbear to catch likely bugs

### PR DESCRIPTION
**Pull request by sciyoshi**
In particular, this will catch mutable default arguments, which are a common antipattern in Python.

### CODE REVIEW

**Comment by botberry** [comment]
Hi, thanks for contributing to Strawberry 🍓!

We noticed that this PR is missing a `RELEASE.md` file. We use that to automatically do releases here on GitHub and, most importantly, to PyPI!

So as soon as this PR is merged, a release will be made 🚀.

Here's an example of `RELEASE.md`:

```markdown
Release type: patch

Description of the changes, ideally with some examples, if adding a new feature.
```

Release type can be one of patch, minor or major. We use [semver](https://semver.org/),so make sure to pick the appropriate type. If in doubt feel free to ask :)

File: strawberry/schema.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 
 class Schema(GraphQLSchema):
-    def __init__(self, query, mutation=None, subscription=None, directives=[]):
```
Comment (patrick91) [inline strawberry/schema.py]: 👍

**Comment by sciyoshi** [comment]
It should be automatically loaded as part of flake8.

**Comment by sciyoshi** [comment]
Should have been added as a dev dependency however, hang on...

**Comment by sciyoshi** [comment]
Rebased and set flake8-bugbear as a dev dependency :+1:

**Pull request merged**